recipes/aaf/all/test_package/example.cpp

```diff
@@ -0,0 +1,12 @@
+#include "AAF.h"
+#include "AAFResult.h"
+
+int main() {
```

Comment:
I would convert the test to a c source and use `NULL` instead.

Reply:
Doing this make the test fails:

```
...
[ 50%] Building C object CMakeFiles/example.dir/example.c.o
In file included from /Users/martin/dev/conan/cci/recipes/aaf/all/test_package/example.c:1:
In file included from /Users/martin/.conan/data/aaf/1.2.0/_/_/package/1272de91de4260534c0c50b473b3ce1b319b417c/include/AAF.h:44:
In file included from /Users/martin/.conan/data/aaf/1.2.0/_/_/package/1272de91de4260534c0c50b473b3ce1b319b417c/include/AAFCOMPlatform.h:42:
/Users/martin/.conan/data/aaf/1.2.0/_/_/package/1272de91de4260534c0c50b473b3ce1b319b417c/include/AAFCOMPlatformTypes.h:133:7: error:
      unknown type name 'virtual'
      STDMETHOD(QueryInterface) (THIS_ REFIID riid, LPVOID F...
      ^
/Users/martin/.conan/data/aaf/1.2.0/_/_/package/1272de91de4260534c0c50b473b3ce1b319b417c/include/AAFCOMPlatformTypes.h:109:35: note:
      expanded from macro 'STDMETHOD'
# define STDMETHOD(method)        virtual HRESULT STDCALL method
                                  ^
/Users/martin/.conan/data/aaf/1.2.0/_/_/package/1272de91de4260534c0c50b473b3ce1b319b417c/include/AAFCOMPlatformTypes.h:133:17: error:
      expected ';' at end of declaration list
      STDMETHOD(QueryInterface) (THIS_ REFIID riid, LPVOID F...
                ^
/Users/martin/.conan/data/aaf/1.2.0/_/_/package/1272de91de4260534c0c50b473b3ce1b319b417c/include/AAFCOMPlatformTypes.h:134:7: error:
      unknown type name 'virtual'
      STDMETHOD_(ULONG,AddRef) (THIS)  PURE;
      ^
...
```

Reply:
Ok. The interface looked like c.

Reply:
Ok just changing `nullptr` to `NULL` seems to works (247a9bc7)

Reply:
I was asking for NULL, because c doesn't have nullptr.
By using `NULL` instead of `nullptr`, perhaps aaf doesn't need c++11